pylint/checkers/misc.py

```diff
@@ -159,16 +159,15 @@ def process_tokens(self, tokens):
                     except PragmaParserError:
                         # Printing useful information dealing with this error is done in the lint package
                         pass
-                    if set(values) & set(self.linter.config.notes):
-                        continue
-                    self.linter.disable("fixme", scope="module", line=comment.start[0])
```

Comment:
This test was always false, so we never got a `continue` that would prevent warnings. It's a bit of a silly test: `values` will always be `fixme`, the msgid. We're testing if the msgid in the disable, i.e. the `pylint: disable=fixme` is in `--notes`, which defaults to `FIXME, XXX, TODO`. Not only should we not be checking that (?!), because you should be able to disable the fixme msg no matter what comments you are searching for, we certainly shouldn't do a case-sensitive comparison.